app/ui/browser-modern/sagas/page-sagas.js

```diff
@@ -77,6 +81,32 @@ export function* destroyPageSession({ id, withUI }) {
   }
 }
 
+function* forkPage(pageId, historyIndex) {
+  const currentPageIndex = yield select(PageSelectors.getPageIndexById, pageId);
+  const historyURLs = yield select(PageSelectors.getPageHistoryURLs, pageId);
```

Comment:
Two yields in a row -> is it possible for the page to have closed (e.g., by `window.close()`) and thus not exist by the time we come to get the history URLs?